examples/api-call/api_call.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
```

Comment:
Example name could be more related to outbound network. I was confused by it at first. And api call sounds like some call to yagna API or something like this

Reply:
I named it this way, because I thought "outbound network" will be an expression recoginzable only by us. Majority usage of outbound network will involve requests to an an external REST API, so I thought it would be better to have it named in a way where user trying to do this will find it instantly. Having said that it looks weird alongside `low-level-api` example.

Reply:
After discussion we decided to name it: `external-api-request`

Reply:
Changed it.